generated/nidaqmx/system/physical_channel.py

```diff
@@ -619,15 +620,18 @@ def configure_teds(self, file_path=""):
         channel that you performed in MAX.
 
         Args:
-            file_path (Optional[str]): Is the path to a Virtual TEDS
-                data sheet that you want to associate with the physical
-                channel. If you do not specify anything for this input,
-                this function attempts to find a TEDS sensor connected
-                to the physical channel.
+            file_path (Optional[pathlib.PurePath]): Is the path to a
+                Virtual TEDS data sheet that you want to associate with
+                the physical channel. If you do not specify anything for
+                this input, this function attempts to find a TEDS sensor
+                connected to the physical channel.
         """
+        if not file_path.name:
+            file_path = "" # type: ignore
```

Comment:
Is this about how `str(pathlib.PurePath(""))` evaluates to `"."`? That suggests that the default should be `None` rather than `pathlib.PurePath("")`.

Use `Optional[T]` in the type hint to indicate that the parameter is allowed to be `None`. 

Note that this is completely orthogonal to whether the parameter uses default args. `def f(x: int = 3)` uses default args but does not allow `None`. `def f(x: Optional[int])` allows `None` but does not use default args.

For `Union[X, Y]`, you can use either `Optional[Union[X, Y]]` or `Union[X, Y, None]`. They should be equivalent.